lambda/MAS/Controllers/ContentController.cs

```diff
@@ -79,6 +86,23 @@ public async Task<IActionResult> PutAsync([FromBody] Item item)
                     return Validate(itemXmlResponseStatus, _logger);
                 }
 
+                //Cache invalidation
+                var paths = new List<string>()
+                {
+                    _awsConfig.ServiceURL + "/sitemap.xml",
```

Comment:
Not sure we need service URL on the start of these, normally the invalidation rules are just /* etc